The job **`aarch64-gnu-llvm-19-1`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/17141006657/job/48628310383) [(plain enhanced)](https://triage.rust-lang.org/gha-logs/rust-lang/rust/48628310383) [(plain)](https://github.com/rust-lang/rust/commit/3501e4f3f2a0501d353fb97ca823443203095682/checks/48628310383/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
Testing option-type-signatures.js ... OK
Testing osstring-to-string.js ... OK
Testing parser-bindings.js ... OK
Testing parser-errors.js ... FAILED
[ query `-> *`]==> Different values for field `foundElems`:
EXPECTED: `0`
RESULT:   `1`
[ query `-> *`]==> RESULT has extra value in array from field `returned` (position 0): `{"name":"pointer","id":null,"fullPath":["pointer"],"pathWithoutLast":[],"pathLast":"pointer","normalizedPathLast":"pointer","generics":[],"bindings":{},"typeFilter":"primitive","bindingName":null}` compared to EXPECTED
[ query `-> *`]==> Different values for field `error`:
EXPECTED: `"Unexpected `*` after ` ` (not a valid identifier)"`
RESULT:   `null`
Testing parser-filter.js ... OK
Testing parser-generics.js ... OK
Testing parser-hof.js ... OK
Testing parser-ident.js ... OK
Testing parser-literal.js ... OK
---
Testing vec-new.js ... OK
Testing vec-type-signatures.js ... OK
Testing write.js ... OK
Bootstrap failed while executing `--stage 2 test --skip compiler --skip src`
Command `/usr/bin/node /checkout/src/tools/rustdoc-js/tester.js --crate-name std --resource-suffix 1.91.0 --doc-folder /checkout/obj/build/aarch64-unknown-linux-gnu/doc --test-folder /checkout/tests/rustdoc-js-std` failed with exit code 1
Created at: src/bootstrap/src/core/build_steps/test.rs:957:27
Executed at: src/bootstrap/src/core/build_steps/test.rs:990:17

Command has failed. Rerun with -v to see more details.
Build completed unsuccessfully in 0:48:17
  local time: Thu Aug 21 23:47:21 UTC 2025
  network time: Thu, 21 Aug 2025 23:47:21 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/tests/rustdoc-js-std/parser-errors.js
+++ b/tests/rustdoc-js-std/parser-errors.js
@@ -16,14 +16,6 @@ const PARSED = [
         error: "Found generics without a path",
     },
     {
-        query: '-> *',
-        elems: [],
-        foundElems: 0,
-        userQuery: "-> *",
-        returned: [],
-        error: "Unexpected `*` after ` ` (not a valid identifier)",
-    },
-    {
         query: 'a<"P">',
         elems: [],
         foundElems: 0,

